mslib/mscolab/file_manager.py

```diff
@@ -104,6 +104,17 @@ def list_operations(self, user):
             })
         return operations
 
+    def is_member(self, u_id, op_id):
+        """
+        op_id: operation id
+        u_id: user-id
+        """
+        # return true only if the user is a member
+        permi = Permission.query.filter_by(u_id=u_id, op_id=op_id).first()
+        if not permi:
```

Comment:
better compare please to None

if perm is not None:

 we abuse boolean here and some other none cpython interpreter can do it wrong